tracee-ebpf/tracee/consts.go

```diff
@@ -14,6 +15,12 @@ type bpfConfig uint32
 // Matches 'MAX_STACK_DEPTH' in eBPF code
 const maxStackDepth int = 20
 
+// Custom KernelConfigOption's to extend kernel_config helper support
+// Add here all kconfig variables used within tracee.bpf.c
+const (
+	CONFIG_ARCH_HAS_SYSCALL_WRAPPER helpers.KernelConfigOption = iota + 1000
```

Comment:
See my other comment in libbpfgo: https://github.com/aquasecurity/libbpfgo/pull/70#discussion_r702408592